### PR TITLE
Docstream

### PR DESCRIPTION
**Pull request by enicolaisen**
not sure what other examples I could add. Where could I find more?

### CODE REVIEW

File: datalad_dataverse/create_sibling_dataverse.py [hunk 1]
```diff
@@ -68,10 +68,30 @@ class CreateSiblingDataverse(Interface):
...
     
+    In order to be able to use this command, a personal access token has to be generated on the Dataverse platform. You can find it by clicking on your name at the top right corner, and then clicking on Api Token>Create Token.
+
     TODO: command doc
     """
 
-    _examples_ = []
+    _examples_ = [
+        dict(text="Use a new sibling on GIN as a common data source that is "
+                  "auto-available when cloning from GitHub",
```
Comment (adswa) [inline datalad_dataverse/create_sibling_dataverse.py:78]: I think the description here is not yet what you want it to be :) You can make it something generic like "Create a dataset sibling in the form of a dataverse dataset"

Comment (enicolaisen) [reply datalad_dataverse/create_sibling_dataverse.py:78]: oh true. thanks

Comment (enicolaisen) [reply datalad_dataverse/create_sibling_dataverse.py]: I changed and pushed

File: datalad_dataverse/create_sibling_dataverse.py [hunk 2]
```diff
@@ -68,10 +68,29 @@ class CreateSiblingDataverse(Interface):
...
     """
 
-    _examples_ = []
+    _examples_ = [
+        dict(text="Create a dataset sibling in the form of a dataverse dataset",
+             code_py="""\
+                 > ds = Dataset('.')
+
+                 # the sibling on Dataverse will be used for data sharing and citing
+                 > ds.create_sibling_dataverse('myrepo', name='dataverse')
```
Comment (adswa) [inline datalad_dataverse/create_sibling_dataverse.py:82]: what both commands need in this example is a `url` as the first parameter to the command. You can use ``demo.dataverse.org``, e.g., like this:
```
ds.create_sibling_dataverse(url='https://demo.dataverse.org', name='dataverse')
```

The command line example should also replace ``myrepo`` with ``demo.dataverse.org``

Comment (enicolaisen) [reply datalad_dataverse/create_sibling_dataverse.py]: corrected

File: datalad_dataverse/create_sibling_dataverse.py [hunk 3]
```diff
@@ -68,10 +68,29 @@ class CreateSiblingDataverse(Interface):
...
+    In order to be able to use this command, a personal access token has to be generated on the Dataverse platform. You can find it by clicking on your name at the top right corner, and then clicking on Api Token>Create Token.
+
     TODO: command doc
     """
 
-    _examples_ = []
+    _examples_ = [
+        dict(text="Create a dataset sibling in the form of a dataverse dataset",
+             code_py="""\
+                 > ds = Dataset(url='https://demo.dataverse.org')
```
Comment (adswa) [inline datalad_dataverse/create_sibling_dataverse.py:79]: Sorry, I didn't mean to suggest that this part should be changed :) Lets keep it as you previously had it:

```suggestion
                 > ds = Dataset('.')
```

File: datalad_dataverse/create_sibling_dataverse.py [hunk 4]
```diff
@@ -68,10 +68,29 @@ class CreateSiblingDataverse(Interface):
...
+
+                 # the sibling on Dataverse will be used for data sharing and citing
+                 > ds.create_sibling_dataverse(url='https://demo.dataverse.org', name='dataverse')
+
+                 """,
+             code_cmd="""\
+                 % datalad create-sibling-dataverse demo.dataverse.org -s dataverse
+
+                 # the sibling on Dataverse will be used for data sharing and citing
+                 % datalad create-sibling-dataverse demo.dataverse.org -s dataverse
```
Comment (adswa) [inline datalad_dataverse/create_sibling_dataverse.py:89]: This example has the command twice, shall we delete the first one?
```suggestion
                 # the sibling on Dataverse will be used for data sharing and citing
                 % datalad create-sibling-dataverse demo.dataverse.org -s dataverse
```

Comment (enicolaisen) [reply datalad_dataverse/create_sibling_dataverse.py]: yep, sorry. Done

File: datalad_dataverse/create_sibling_dataverse.py [hunk 5]
```diff
@@ -68,10 +68,26 @@ class CreateSiblingDataverse(Interface):
...
+
     TODO: command doc
     """
 
-    _examples_ = []
+    _examples_ = [
+        dict(text="Create a dataset sibling in the form of a dataverse dataset",
+             code_py="""\
+                 > ds = Dataset('.')
+
```
Comment (adswa) [inline datalad_dataverse/create_sibling_dataverse.py:80]:
```suggestion
```

**Comment by adswa** [comment]
Thanks for the examples and edits!

**Comment by adswa** [comment]
I'll merge now, and we can extend them later

**Pull request merged**